python/ray/serve/schema.py

```diff
@@ -287,18 +161,21 @@ def get_user_configured_option_names(self) -> Set[str]:
         }
 
 
-def _deployment_info_to_schema(name: str, info: DeploymentInfo) -> DeploymentSchema:
-    """Converts a DeploymentInfo object to DeploymentSchema.
+@PublicAPI(stability="stable")
+class ReadServeDeploymentModel(BaseDeploymentModel):
```

Comment:
I would prefer using a more readable name rather than something so explicit, as most examples I've seen have followed a similar pattern - lmk what you think!

Reply:
I'm okay with keeping it consistent. We can stick with the current name.